src/openms/source/CHEMISTRY/ModifiedPeptideGenerator.cpp

```diff
@@ -235,131 +227,48 @@ namespace OpenMS
         // Term specificity is ANYWHERE on the peptide, C_TERM or N_TERM
         // (currently no explicit support in OpenMS for protein C-term and
         // protein N-term)
+        // TODO This is not true anymore!
         const ResidueModification::TermSpecificity& term_spec = v->getTermSpecificity();
+        // TODO why are we not checking for existing modifications like in the N/C Term case above??
```

Comment:
in line 209 it is checked if the residue is modified

Reply:
true. removed comment.